test/basic.html

```diff
@@ -376,11 +390,50 @@
         assert.equal(input.value, 'bar');
         assert.isTrue(checkbox1.checked);
         assert.isFalse(checkbox2.checked);
+        assert.equal(paperInput.value, 'zug');
+        assert.equal(paperInput.inputElement.value, 'zug');
+        assert.equal(paperTextarea.value, 'zog');
+        assert.equal(paperTextarea.inputElement.textarea.value, 'zog');
         done();
       });
 
       form.reset();
+    });
+
+    test('form restores the default values if no changes were made', function(done) {
```

Comment:
Nope, the previous tests takes inputs, makes changes to them, then restores them. This test takes inputs and restores them immediately. You'd be surprised how one failed and the other didn't :)